tests/e2e/operation/multi.go

```diff
@@ -0,0 +1,219 @@
+// +build e2e
+
+//
+// Copyright (C) 2019-2021 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package operation
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"github.com/vdaas/vald/apis/grpc/v1/payload"
+)
+
+func (c *client) MultiSearch(t *testing.T, ctx context.Context, ds Dataset) error {
+	client, err := c.getClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	cfg := &payload.Search_Config{
+		Num:     3,
+		Radius:  -1.0,
+		Epsilon: 0.1,
+	}
+
+	reqs := make([]*payload.Search_Request, 0, len(ds.Test))
+	for _, v := range ds.Test {
+		reqs = append(reqs, &payload.Search_Request{
+			Vector: v,
+			Config: cfg,
+		})
+	}
+
+	req := &payload.Search_MultiRequest{
+		Requests: reqs,
+	}
+	_, err = client.MultiSearch(ctx, req)
+	if err != nil {
+		return err
+	}
```

Comment:
I suggest to check the result length and also the distance contains 0 (exact result). 
Same as below functions.
```suggestion
	res, err = client.MultiSearch(ctx, req)
	if err != nil {
		return err
	}
	if len(res.Results) != len(ds.Test) {
	        return errors.New("response length does not match with dataset")
	}
	if res. Results[0].Distance != 0 {
	        return errors.New("first result is not 0")
	}
```

Reply:
Thanks, but it should be like the following.
```suggestion
	res, err := client.MultiSearch(ctx, req)
	if err != nil {
		return err
	}
	
	if len(res.GetResponses()) != len(ds.Test) {
		return errors.New("number of responses does not match with sent requests")
	}
```

The first element of each result doesn't always have 0 distance.

Reply:
thank you. should we check the result contains 0 distance element?

Reply:
No. in almost cases, the results don't contain 0 distance element.